vcpkg-init/vcpkg-init.ps1

```diff
@@ -126,162 +64,77 @@ function bootstrap-vcpkg {
   }
 
   & $VCPKG bootstrap-standalone
-
-  $PATH = $ENV:PATH
-  $ENV:PATH="$VCPKG_ROOT;$PATH"
-
-  Set-Content -Path $VCPKG_VERSION_MARKER -Value "$VCPKG_INIT_VERSION`n" -Force -NoNewline
-
-  z-vcpkg-debug "Bootstrapped vcpkg: ${VCPKG_ROOT}"
-
-  if( -not ( test-path $VCPKG_SCRIPT )) {
-    Write-Error "ERROR! Bootstrapping vcpkg failed."
-    return $false
+  if(-Not $?) {
+    Write-Error "Bootstrap failed."
+    return $False
   }
-  return $true
-}
 
-if( -not (bootstrap-vcpkg )) {
-  write-error "Unable to install vcpkg."
-  throw "Installation Unsuccessful."
+  Write-Host "Bootstrapped vcpkg: $env:VCPKG_ROOT"
```

Comment:
This no longer writes `VCPKG_VERSION_MARKER` -- how does it avoid re-downloading vcpkg every time?